[2021-12-29 15:16:23,400] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.create_fact_table_query 2021-12-28T15:13:12.251660+00:00 [queued]>
[2021-12-29 15:16:23,418] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.create_fact_table_query 2021-12-28T15:13:12.251660+00:00 [queued]>
[2021-12-29 15:16:23,419] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 15:16:23,419] {taskinstance.py:1095} INFO - Starting attempt 1 of 2
[2021-12-29 15:16:23,419] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 15:16:23,430] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): create_fact_table_query> on 2021-12-28T15:13:12.251660+00:00
[2021-12-29 15:16:23,434] {standard_task_runner.py:52} INFO - Started process 16855 to run task
[2021-12-29 15:16:23,437] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cleansing_dag', 'create_fact_table_query', '2021-12-28T15:13:12.251660+00:00', '--job-id', '548', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmp4qtsd6nq', '--error-file', '/tmp/tmp6mmf675n']
[2021-12-29 15:16:23,438] {standard_task_runner.py:77} INFO - Job 548: Subtask create_fact_table_query
[2021-12-29 15:16:23,476] {logging_mixin.py:109} INFO - Running <TaskInstance: cleansing_dag.create_fact_table_query 2021-12-28T15:13:12.251660+00:00 [running]> on host 34521d0e1651
[2021-12-29 15:16:23,528] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=create_fact_table_query
AIRFLOW_CTX_EXECUTION_DATE=2021-12-28T15:13:12.251660+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-28T15:13:12.251660+00:00
[2021-12-29 15:16:25,387] {python.py:151} INFO - Done. Returned value was: None
[2021-12-29 15:16:25,400] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=create_fact_table_query, execution_date=20211228T151312, start_date=20211229T151623, end_date=20211229T151625
[2021-12-29 15:16:25,457] {local_task_job.py:151} INFO - Task exited with return code 0
